lblod/app-lokaal-mandatenbeheer
# fix uris for nested instances

## Description

Fix the uris generated for nested instances in mandataris and persoon forms
Before:
![image](https://github.com/user-attachments/assets/c3aa0b21-cb22-4340-ac02-8b6e5e740adc)

After:
![image](https://github.com/user-attachments/assets/8ec08336-6be2-4569-9381-1276445b508d)

## How to test

Create/edit person/mandataris using this pr (restart form-content service) and see that the generated uris are not form-based
